src/common/MovieTile/styled.js

```diff
@@ -154,20 +198,29 @@ export const Rate = styled.div`
     font-size: 16px;
     padding: 0 12px;
 
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobileMin}px) {
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobileVertical}px) {
```

Comment:
hmm, please check again if the names of the breakpoints match ;) because here you can see that you have changed the breakpoint to "mobileVertical", and in the theme.js file completely different names of these points, look here: 

```
        tablet: 992,
        mobileMax: 767,
```